recipes/arraymorph/meta.yaml

```diff
@@ -0,0 +1,72 @@
+package:
+  name: arraymorph
+  version: 0.1.1
+
+source:
+  url: https://github.com/ICICLE-ai/ArrayMorph/archive/refs/tags/v0.1.1.tar.gz
+  sha256: 833651c4440971cc52469c18d8b8cfa521080a085e09661be5027ecbbae9ddca
+
+build:
+  skip: true  # [not linux64]
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('arraymorph', max_pin='x.x') }}
+  script: |
+    #!/bin/bash
+    # Build from source code
+    cd "${SRC_DIR}/arraymorph/" || exit 1
+    cmake -B ./build -S . -DCMAKE_PREFIX_PATH=$BUILD_PREFIX
+    cd ./build
+    make
+    cd ..
+    # Create target directory
+    mkdir -p ${PREFIX}/lib/arraymorph
+    # Install library
+    cp ./build/src/libarraymorph.so ${PREFIX}/lib/arraymorph/
+    # Install activation/deactivation scripts
+    mkdir -p ${PREFIX}/etc/conda/activate.d
+    mkdir -p ${PREFIX}/etc/conda/deactivate.d
+    cp activate.sh ${PREFIX}/etc/conda/activate.d/arraymorph-activate.sh
+    cp deactivate.sh ${PREFIX}/etc/conda/deactivate.d/arraymorph-deactivate.sh
+    # Make scripts executable
+    chmod +x ${PREFIX}/etc/conda/activate.d/arraymorph-activate.sh
+    chmod +x ${PREFIX}/etc/conda/deactivate.d/arraymorph-deactivate.sh
+  post-link: |
+    #!/bin/bash
+    export HDF5_PLUGIN_PATH=$CONDA_PREFIX/lib/arraymorph/
+    export HDF5_VOL_CONNECTOR=arraymorph
+
+requirements:
+  build:
+    # Linux-specific GCC 9 toolchain
+    - gcc_linux-64 9.*
+    - gxx_linux-64 9.*
```

Comment:
If you really need an old version of GCC, you should still use our compiler templates. Just override the `c_compiler_version` and `cxx_compiler_version` in a local `conda_build_config.yaml`